lib/generator.js

```diff
@@ -212,10 +212,39 @@ const generateReturns = (comments, customTypes, usedTypes) => {
   return [line, '', ...res];
 };
 
-const ESCAPE_MD_REGEXP = /[_[\]]/g;
+const isAlphaNumeric = str => {
+  if (!str) return false;
+  for (const char of str) {
+    const code = char.charCodeAt();
+    if (
+      !(code > 47 && code < 58) &&
+      !(code > 64 && code < 91) &&
+      !(code > 96 && code < 123)
+    ) {
+      return false;
+    }
+  }
+  return true;
+};
+
+const escapeMd = str => {
+  let res = '';
+  for (let i = 0; i < str.length; i++) {
+    if (
+      str[i] === '>' ||
+      str[i] === '[' ||
+      str[i] === ']' ||
+      (str[i] === '_' &&
+        !(isAlphaNumeric(str[i - 1]) && isAlphaNumeric(str[i + 1])))
```

Comment:
This would ignore all non-ascii identifier symbols, i.e. `const func = (a = _µµ_µµ_)`, would that be fine?

Refs: https://www.ecma-international.org/ecma-262/9.0/index.html#sec-identifier-names
https://unicode.org/cldr/utility/list-unicodeset.jsp?a=%5B%3AID_Start%3A%5D&g=&i=
https://unicode.org/cldr/utility/list-unicodeset.jsp?a=%5B%3AID_Continue%3A%5D&g=&i=

/cc @belochub

Reply:
IMO we shouldn't support non-ascii identifier symbols. Moreover, our eslint config won't even [allow](https://github.com/metarhia/eslint-config-metarhia/blob/0c6bd58f7167df1ad59114689cc06737db543d3f/rules/stylistic-issues.js#L25) such identifiers.

Reply:
@SemenchenkoVitaliy the rule you linked doesn't check if the identifier contains Unicode characters with code points values higher than 0x7F, it just checks that the identifier is in camel case, and it is possible to use camel case in other languages. As for using underscores in the middle of the identifier with this rule, they can be used when everything else is in uppercase (as in constants), for example:
```js
const ПРИВІТ_СВІТЕ = 'Hello, world!';
```